event-svc/benches/sqlite_store.rs

```diff
@@ -8,10 +8,10 @@ use ceramic_event::unvalidated::{
 };
 use criterion2::{criterion_group, criterion_main, BatchSize, Criterion};
 use ipld_core::ipld::Ipld;
-use itertools::Itertools;
+use itertools::Itertools as _;
```

Comment:
why _

Reply:
There's really not point to this change.. it was showing up as unneeded so I deleted it but it *was* needed so my IDE added it back with _.. but generally traits only need to be in scope, so using `as _` can be nice to avoid name collisions (plus it makes it clear it's a trait by convention) but it's same same.